src/gui/gui_nation_picker.hpp

```diff
@@ -664,15 +664,36 @@ class start_game_button : public button_element_base {
 			}
 			text::close_layout_box(contents, box);
 		}
+
+		auto box = text::open_layout_box(contents, 0);
+		text::add_to_layout_box(state, contents, box, std::string_view("在标准状况下 medžuslovjansky jezyk ?"));
```

Comment:
I am guessing that this text is here for debugging, and you have left it in accidentally?